day_6/exercises/burrito.js

```diff
@@ -16,12 +16,23 @@ class Burrito {
     this.base = base;
     this.toppings = toppings;
   }
+  addTopping(topping){
+    this.toppings.push(topping);// adds a push method to and a topping
+  }
+  removeTopping(topping){
+    var indextoRemove = this.toppings.indexOf(topping)
+    this.toppings.splice(indextoRemove,1); //uses the .splice to remove the index of salsa
+  }
+  changeProtein(){
+    (this.protein = "Chicken"); // assings Chicken as value
```

Comment:
This `changeProtein` method is automatically assigning "chicken" as the new protein. What about if you passed in the new protein as an argument to the method like so?
```
changeProtein(newProtein) {
      this.protein = newProtein;
}
```